projects/npm-tools/packages/npm-scripts/src/utils/tweakWebpackConfig.js

```diff
@@ -12,16 +12,28 @@ const createTempFile = require('./createTempFile');
 const getMergedConfig = require('./getMergedConfig');
 const parseBnd = require('./parseBnd');
 
-const CORE_REMOTES = ['frontend-js-react-web'];
-const CORE_SHARES = ['react'];
+const CORE_REMOTES = ['frontend-js-react-web', 'frontend-taglib-clay'];
+const CORE_SHARES = [
+	'@clayui/icon',
+	'classnames',
+	'formik',
+	'prop-types',
+	'react',
+	'react-dnd',
+	'react-dnd-html5-backend',
+	'react-dom',
+];
```

Comment:
This will be moved to a better place once it's clear what we need. It is the equivalent of the `imports` section of the default preset.